checkout-extension/shopify.extension.toml.liquid

```diff
@@ -1,35 +1,51 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
```

Comment:
I felt it was useful to offer a link to each of the key concepts being promoted by the template config file.

Reply:
Thank you! I had a draft item in the project for this, if you want to mark it as done: https://github.com/orgs/Shopify/projects/5920/views/1?pane=issue&itemId=33989493